app/assets/javascripts/components/buttons/form-button-component.js

```diff
@@ -0,0 +1,47 @@
+ManageIQ.angular.app.component('formButtonComponent', {
```

Comment:
Renaming the component correctly can go a long way.

The new name `form-button-component` indicates that this is really a form-based component (that has buttons), and not to be confused with a button component.